pkg/cri/server/snapshots.go

```diff
@@ -49,27 +52,46 @@ func newSnapshotsSyncer(store *snapshotstore.Store, snapshotter snapshot.Snapsho
 	}
 }
 
-// start starts the snapshots syncer. No stop function is needed because
-// the syncer doesn't update any persistent states, it's fine to let it
-// exit with the process.
+// start starts the snapshots syncer.
 func (s *snapshotsSyncer) start() {
-	tick := time.NewTicker(s.syncPeriod)
+	if s.syncTicker != nil {
```

Comment:
do we have any usage about `syncTicker` here? Looks like we don't need that field here~

Reply:
I am not sure about usage, but if we need to shut it down, it's necessary to put it into a struct field.